kolibri/plugins/learn/assets/src/views/content-card-group-grid/index.vue

```diff
@@ -26,7 +26,7 @@
       :title="content.title"
       :thumbnail="content.thumbnail"
       :class="{'grid-item': true, 'mobile': isMobile}"
-      :kind="content.kind"
+      :kind="content.id === content.channel_id ? channelKind : content.kind"
```

Comment:
Why isn't `content.kind` equal to CHANNEL to begin with for channels?

Reply:
Are you referring to why it's kind isn't CHANNEL in the back-end?

Reply:
I can also move this logic to the _topicState(), but that doesn't address the fact that it comes with kind set to topic.

Reply:
@jonboiser Moved logic to TopicState

Reply:
Okay, that sounds like a good idea to remove that logic from the component.

So channels have kind 'topic' and the only way to check if a 'topic' is a channel is to check the channel and node id are the same?

Reply:
Looks like it:

![image](https://user-images.githubusercontent.com/7193975/30394664-9a97e714-9878-11e7-93d6-e97b985306e3.png)

Reply:
I believe that channels have a "root topic node", and that's the thing we actually interact with in the API.

Reply:
It's always seemed weird to me, too. Maybe there's a possibility to refactor.